coos.html

```diff
@@ -511,49 +523,51 @@ <h3>Base GSBPM model</h3>
+				
+				<p>Three individuals of <code>ActivityCategory</code> are define in the ontology to model the types of overarching activities provided by GSBPM</p>
 
 				<pre class="example nohighlight" title="Examples of GSBPM overarching activity">
-				&lt;http://id.unece.org/activities/overarchingActivity/0.2&gt; a coos:OverarchingActivity ;
+				&lt;http://id.unece.org/activities/overarchingActivity/0.2&gt; a coos:ActivityCategory ;
 					skos:notation "0.2"	;
 					skos:prefLabel "Metadata Management"@en .
 				</pre>
 
-				<p>As with GSBPM, COOS defines individuals for each GAMSO activity and activity area, and the model itself is a <code>skos:ConceptScheme</code> with all four activity areas having <code>skos:broader</code> links to it and <code>skos:narrower</code> links to the activities they contain. At the top of the scheme, an instance of <code>Activity</code> represents the whole activity of statistical organizations and is named <code>officialStatistics</code>.</p>
+				<p>As with GSBPM, COOS defines individuals for each GAMSO activity and activity area, and the model itself is a <code>skos:ConceptScheme</code> with all four activity areas having <code>skos:broader</code> links to it and <code>skos:narrower</code> links to the categories of activity they contain. At the top of the scheme, an instance of <code>ActivityCategory</code> represents the whole activity of statistical organizations and is named <code>officialStatistics</code>.</p>
+
+				<pre class="example nohighlight" title="Examples of GAMSO activity area and activity category">
+				&lt;http://id.unece.org/activities/activityArea/1&gt; a coos:ActivityArea ;
+					skos:notation "1" ;
+					skos:prefLabel "Strategy and Leadership"@en .
 
-				<p>Finally, two properties are needed to describe how a GAMSO <code>Activity</code> relates to a GSBPM <code>StatisticalProductionActivity</code>:</p>
+				&lt;http://id.unece.org/activities/activity/2.3&gt; a coos:ActivityCategory ;
+					skos:notation "2.3" ;
+					skos:prefLabel "Monitor Capability Improvements"@en .
+				</pre>
+				
+				<p>Finally, two properties are needed to describe how an <code>Activity</code> classified with GAMSO relates to a <code>StatisticalProductionActivity</code> classified with GSBPM:</p>
 
 				<ul>
-					<li><code>supports</code>: an <code>Activity</code>, for example a GAMSO individual, helps to perform a <code>StatisticalProductionActivity</code>, e.g. a sub-process;</li>
-					<li><code>uses</code>: conversely, a statistical production activity will require one or several GAMSO activities in order to function.</li>
+					<li><code>supports</code>: an <code>Activity</code>, for example of type a GAMSO individual, helps to perform a <code>StatisticalProductionActivity</code>, e.g. a sub-process;</li>
```

Comment:
"for example of type a GAMSO individual" should probably be replaced by "for example, a GAMSO individual"

Reply:
I removed the reference to Gamso